assertj-tests/assertj-integration-tests/assertj-guava-tests/src/test/java/org/assertj/tests/guava/api/TableAssert_isNotEmpty_Test.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2024 the original author or authors.
+ */
+package org.assertj.tests.guava.api;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.assertj.guava.api.Assertions.assertThat;
+
+import org.assertj.guava.api.TableAssert;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Maciej Kucharczyk
+ */
+class TableAssert_isNotEmpty_Test extends TableAssertBaseTest {
+
+  @Test
+  void should_pass_if_actual_is_not_empty() {
+    assertThat(actual).isNotEmpty();
+  }
+
+  @Test
+  void should_fail_if_actual_is_null() {
+    // GIVEN
+    actual = null;
+    // WHEN
+    Throwable throwable = catchThrowable(() -> assertThat(actual).isNotEmpty());
+    // THEN
+    assertThat(throwable).isInstanceOf(AssertionError.class)
+      .hasMessage(actualIsNull());
```

Comment:
We favor BDD style assertions for our own tests, i.e., :

```suggestion
    then(throwable).isInstanceOf(AssertionError.class)
      .hasMessage(actualIsNull());
```

From `org.assertj.core.api.BDDAssertions.then`